### ISSUE TITLE
Publishing to Maven

### ISSUE THREAD

**Issue by EasyG0ing1** [adolfintel/WaifUPnP#19]
Hello,

I'm assuming based on issue #8 that it won't be a problem if I publish this library on Maven.  However, I'll have to publish under my domain. I will, of course, maintain your copyright info as well as your name throughout the library.

I'm also thinking about publishing a modular version for Java 9+ though it might be enough to just include an automatic module name in the manifest file... certainly would be enough for most devs anyway... 

It's an awesome library that just works, and it works well.

Thanks.

**Comment by adolfintel** [comment]
Sure! Go ahead

**Comment by EasyG0ing1** [comment]
@adolfintel I've already got it ready to publish, but I'm looking at the pull requests and debating whether or not to work them into the project first. I suppose I could just publish it as is and work on it after that.  I kicked into iTerm to run a test package and maven wouldn't run at all ... saying that my JDK path was not right ... well after some digging, it turns out it was because of some corruption in a folder within the MacOS Library folder and instead of trying to repair it, I'm just gonna do a clean install of MacOS and restore from TimeMachine... weird problem though ... it kicks out random paths to the screen when I simply CD into the JavaVirtualMachine folder. Never seen anything like it before.
